treeherder/webapp/api/logslice.py

```diff
@@ -79,8 +83,7 @@ def list(self, request, project, jm):
                 return Response(lines)
 
             except Exception as e:
-                logging.error(e)
-                raise ResourceNotFoundException("log file not found")
+                raise
```

Comment:
Hmm, actually, you're right.  I don't know that I've ever done a try... finally without an except as well, tbh.  But the except is superfluous here...  I'll remove...